codec/v2/stablejson/any.go

```diff
@@ -0,0 +1,64 @@
+package stablejson
```

Comment:
could you quickly tldr this? 

Things like this was part of the reason moving away from amino. We didnt want to maintain our own encoding.

Reply:
My goal here is not to change the protojson encoding. I just want something deterministic. I needed it for the orm and now for typed events (not sure determinism is 100% necessary but feel better with it than without). Protojson is explicitly non deterministic.

There is a separate convo to change bytes encoding to hex. I don't actually agree. I think we should stick with the standard. But that's a separate matter.